app/controllers/chargeback_assignment_controller.rb

```diff
@@ -4,108 +4,68 @@ class ChargebackAssignmentController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
-  include Mixins::SavedReportPaging
   include Mixins::GenericSessionMixin
   include Mixins::BreadcrumbsMixin
 
   def self.table_name
     @table_name ||= "chargeback_assignment"
   end
 
-  def tree_select
-    self.x_active_tree = params[:tree] if params[:tree]
-    self.x_node = params[:id]
-    get_node_info(x_node)
-    replace_right_cell
-  end
-
-  def explorer
+  def index
     @breadcrumbs = []
-    @explorer    = true
-    build_accordions_and_trees
-
-    @right_cell_text = _("All Assignments")
-    set_form_locals if @in_a_form
+    title
     session[:changed] = false
-
-    render :layout => "application" unless request.xml_http_request?
-  end
-
-  def set_form_locals
-    @x_edit_buttons_locals = {
-      :action_url   => 'cb_assign_update',
-      :no_cancel    => true,
-      :multi_record => true
-    }
+    @edit = {:new => {}}
```

Comment:
We're not in edit, why these two lines? This whole method seems off to me :confused:

Reply:
Initial screen of CB Assignments is a form with a type drop down so i need this set to render the form correctly, since i store the value of type drop down in `@edit`